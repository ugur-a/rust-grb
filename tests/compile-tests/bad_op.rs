use grb::prelude::*;
mod utils;

fn main() -> grb::Result<()> {
    create_model!(_g, m, x, y, z);
    c!(x + y + 2 * z + 1);
    c!(2 * z);
    c!(2 / z);
    Ok(())
}
