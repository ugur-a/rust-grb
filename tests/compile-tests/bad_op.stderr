error: expression should be a ==, >= or <= comparison
 --> tests/compile-tests/bad_op.rs:6:8
  |
6 |     c!(x + y + 2 * z + 1);
  |        ^^^^^^^^^^^^^^^^^

error: expression should be a ==, >= or <= comparison
 --> tests/compile-tests/bad_op.rs:7:8
  |
7 |     c!(2 * z);
  |        ^^^^^

error: expression should be a ==, >= or <= comparison
 --> tests/compile-tests/bad_op.rs:8:8
  |
8 |     c!(2 / z);
  |        ^^^^^
